pytest_fixtures/component/rh_cloud.py

```diff
@@ -36,6 +36,10 @@ def module_target_sat_insights(request, module_target_sat, satellite_factory):
         iop_settings = settings.rh_cloud.iop_advisor_engine
         script = (iop_settings.setup_script or '').splitlines()
 
+        # Use HTTPS_PROXY to reach container registry for IPv6
+        if settings.server.is_ipv6:
+            satellite.enable_ipv6_system_proxy()
```

Comment:
This helper already takes care of checking the setting value for IPv6
```suggestion
        satellite.enable_ipv6_system_proxy()
```

Reply:
i thought i'd save calling the method unnecessarily, but ok. :)